.github/workflows/e2e-cross-version.yml

```diff
@@ -2,12 +2,31 @@ name: E2E Cross-version Tests
 
 on:
   schedule:
-    - cron: '0 9 * * 0'
+    - cron: '0 9 * * 1-5'
```

Comment:
these test are actually really fast since we're not building metabase. I'd rather run them daily than wonder how they broke in the past week.